Triangulation_3/doc/Triangulation_3/CGAL/Regular_triangulation_cell_base_3.h

```diff
@@ -23,39 +23,57 @@ By default, this parameter is instantiated by
 
 */
 
-template< typename RegularTriangulationTraits_3, typename Cb >
+template< typename Traits, typename Cb >
 class Regular_triangulation_cell_base_3 : public Cb {
 public:
 
 /// \name Types 
 /// @{
-typedef RegularTriangulationTraits_3::Bare_point Bare_point;
+typedef Traits::Point_3 Point_3;
+
+typedef Traits::Weighted_point_3 Point;
```

Comment:
do we really want this `typedef`, where the weight of `Weighted_point_3` becomes somehow hidden?